src/main/kotlin/com/terraformation/backend/accelerator/ParticipantProjectSpeciesService.kt

```diff
@@ -182,7 +183,10 @@ class ParticipantProjectSpeciesService(
         }
 
     val inputStream = ByteArrayInputStream(stream.toByteArray())
-    val filename = "species-list-snapshot-${event.submissionId}-${Instant.now()}.csv"
+    val timestamp =
+        DateTimeFormatter.ofPattern("yyyyMMdd-HHmmss")
+            .format(Instant.now().atZone(ZoneId.of("UTC")))
```

Comment:
Thanks for the info. I was wondering about that, I figured it would be best to not add another dependency for something small, but you make some great points.